### PR TITLE
[docs] Fix DOC-28

### PR DESCRIPTION
**Pull request by ddhodge**
Document Server sequence cache method.

@netlify /preview/reference/configuration/yb-tserver/#ysql-sequence-cache-method

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 1]
```diff
@@ -539,9 +539,21 @@ For details on how online index backfill works, see the [Online Index Backfill](
 
 Default: `false`
 
+##### --ysql_sequence_cache_method
+
+Specifies where to cache sequence values.
+
+Valid values are `connection` and `server`.
+
+This flag requires the YB-TServer `yb_enable_sequence_pushdown` flag to be true (the default). Otherwise, the default behavior will occur regardless of this flag's value.
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/yb-tserver.md:548]: We don't have docs for `yb_enable_sequence_pushdown` ? Asking to link it.

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/reference/configuration/yb-tserver.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/reference/configuration/yb-tserver.md:548]: No we don't. There is a precedent - the other flag that begins with "yb_enable", which is referred to several times but not actually documented

**Review by mislam-yb — APPROVED** [review]
lgtm

**Pull request merged**